doc/manual/expressions/builtins.xml

```diff
@@ -347,7 +347,7 @@ stdenv.mkDerivation { … }
     You can change the cache timeout either on the command line with
     <option>--option tarball-ttl <replaceable>number of seconds</replaceable></option> or
     in the Nix configuration file with this option:
-    <literal>tarball-ttl <replaceable>number of seconds to cache</replaceable></literal>.
+    <literal><xref linkend="conf-tarball-ttl" /> <replaceable>number of seconds to cache</replaceable></literal>.
```

Comment:
Hm, what does this render as?

Reply:
![Screenshot_2019-07-05 Nix Package Manager Guide](https://user-images.githubusercontent.com/76716/60742838-eb892a80-9f3c-11e9-983e-fc18e0cc9bbe.png)

It becomes a link, but (at least, locally) doesn't show a special indication that it is clickable. The term is automatically selected as the link text.